webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue

```diff
@@ -3,14 +3,18 @@
     <div class="showingPeopleText text-sub-title ms-3 d-none d-sm-flex">
       {{ $t('peopleList.label.peopleCount', {0: peopleCount}) }}
     </div>
+    <span>
+      <v-icon v-if="showMobileFilter && isMobile" left size="20" @click="mobileFilter">fa-arrow-left</v-icon>
+    </span>
```

Comment:
Use classes for mobile breakpoints to make UI smoother and add the `v-if` or `v-show` in parent element.

```suggestion
    <span  v-show="showMobileFilter" class="hidden-sm-and-up">
      <v-icon left size="20" @click="mobileFilter">fa-arrow-left</v-icon>
    </span>
```

Reply:
fixed